src/inline-contracts/translations/Messages_fr_FR.json

```diff
@@ -0,0 +1,7 @@
+{
+  "ovh_inline_contracts_label_unique": "Je reconnais avoir bien pris connaissance des {{ contract }}.",
```

Comment:
Wouldn't be better to handle the `.` / `,` directly in HTML with your `$first/$last` conditions ?

Reply:
Not sure about that because for instance the spacing could be different depending on the language